daprdocs/content/en/operations/support/support-release-policy.md

```diff
@@ -69,6 +70,7 @@ General guidance on upgrading can be found for [self hosted mode]({{<ref self-ho
 |                          |                 1.6.0 |                    1.7.4 |
 | 1.6.0                    |                   N/A |                    1.6.2 |
 | 1.6.0                    |                   N/A |                    1.7.4 |
+| 1.7.0 to 1.7.4           |                   N/A |                    1.8.0 |
 
 ## Feature and deprecations
 There is a process for announcing feature deprecations.  Deprecations are applied two (2) releases after the release in which they were announced. For example Feature X is announced to be deprecated in the 1.0.0 release notes and will then be removed in 1.2.0.
```

Comment:
From JS-SDK's 2.3.0 release, we have also deprecated the older NPM package. This has already been announced via different social handles and documentation. Should we add it here as well? More details are here https://github.com/dapr/js-sdk/issues/259

/cc @XavierGeerinck

Reply:
@shubham1172 - Wouldn't this be more suitable in the JS SDK dosc?

Reply:
It is already there, but I just saw a feature and deprecation section with Java SDK announcements too. I wonder if it's a common practice to add stuff here for other SDKs too.